src/app/_components/NewFooter.tsx

```diff
@@ -0,0 +1,231 @@
+    label: 'Stacking',
+    href: '/stacking',
+  },
+  {
+    // This page is not implemented yet
+    label: 'Mempool',
+    href: '/mempool',
+  },
+  {
+    label: 'Signers',
+    href: '/signers',
+  },
+  {
+    label: 'Tokens',
+    href: '/tokens',
+  },
+  {
+    // This page is not implemented yet
+    label: 'NFTs',
+    href: '/nfts',
+  },
+  {
+    // This page is not implemented yet
+    label: 'Analytics',
+    href: '/analytics',
+  },
+  {
+    label: 'Search',
+    href: '/search',
+  },
+];
+
+const leftSideLinks: Link[] = [
+  {
+    label: 'Sandbox',
+    href: '/sandbox/deploy',
+  },
+  {
+    label: 'Status Center',
+    href: 'https://status.hiro.so/',
+  },
+  {
+    label: 'Support',
+    href: '/support',
+  },
+];
+
+const xPadding = 8;
+
+const TopFooterContent = () => {
+  return (
+    <>
+      <Box hideBelow="lg" className="top-footer-content-hide-below-lg">
+        <Flex justifyContent="space-between">
+          <Flex gap={4}>
+            {rightSideLinks.map(link => (
+              <ExplorerLink key={link.label} href={link.href} fontWeight="medium" fontSize="xs">
+                {link.label}
+              </ExplorerLink>
+            ))}
+          </Flex>
+
+          <Flex gap={4}>
+            {leftSideLinks.map(link => (
+              <ExplorerLink key={link.label} href={link.href} fontWeight="medium" fontSize="xs">
+                {link.label}
+              </ExplorerLink>
+            ))}
+          </Flex>
+        </Flex>
+      </Box>
+      <Box hideFrom="lg" className="top-footer-content-hide-from-lg">
+        <Grid templateColumns={['repeat(2, 1fr)', 'repeat(2, 1fr)', 'repeat(3, 1fr)']} gap={4}>
+          {rightSideLinks.concat(leftSideLinks).map(link => (
+            <ExplorerLink href={link.href} fontWeight="medium" fontSize="xs">
+              {link.label}
+            </ExplorerLink>
+          ))}
+        </Grid>
+      </Box>
+    </>
+  );
+};
+
+const BottomFooterContent = () => {
+  return (
+    <Flex justifyContent="space-between" alignItems="center" flexWrap="wrap" rowGap={3}>
+      <BottomLinks />
+      <Link href="https://stacks.co">
+        <Icon h={3.5} w={18}>
+          <StacksNameAndLogoIcon color="var(--stacks-colors-stacks-name-and-logo)" />
+        </Icon>
+      </Link>
+    </Flex>
+  );
+};
+
+const BottomLinks = () => {
+  return (
+    <Flex gap={6} flexWrap="wrap" rowGap={3}>
+      <Flex gap={1.5} alignItems="center" flexWrap="nowrap">
+        <HiroIcon h={4} w={4} color="var(--stacks-colors-icon-tertiary)" />
+        <Text fontSize="xs" color="textSecondary" whiteSpace="nowrap">
+          This Stacks Explorer is built and maintained by{' '}
+          <Link
+            href="https://www.hiro.so"
+            textDecoration="underline"
+            color="textSecondary"
+            _hover={{
+              color: 'textInteractiveHover',
+            }}
+          >
+            Hiro
+          </Link>
+        </Text>
+      </Flex>
+      <Link
```

Comment:
I suggest extracting this link as a component since they all use the same props aside from href and children